src/pages/sensores/Sensores.js

```diff
@@ -1,8 +1,21 @@
+import { ContentHeader } from 'components';
 import classes from './Sensores.module.css';
 
 function Sensores() {
+
+  const onCadastrarNovoClick = () => {
+    console.log('onCadastrarNovoSensorClick');
+  };
+
   return (
-    <div className={classes.pageContainer}>This is the /sensores page</div>
+    <div className={classes.pageContainer}>
+      <ContentHeader
+         title="Sensores"
+         buttonLabel="Cadastrar novo"
+         searchLabel="Pesquisar por sensor"
+         searchPlaceholder="Nome, Localização ou Patrimônio"                
+      />
+    </div>
```

Comment:
```suggestion
    <div className={classes.pageContainer}>
      <ContentHeader
         title="Sensores"
         buttonLabel="Cadastrar novo"
         searchLabel="Pesquisar por sensor"
         searchPlaceholder="Nome, Localização ou Patrimônio"              
         onButtonClick={onCadastrarNovoClick}           
      />
    </div>
```
Acho que você já pode mandar essa função pra dentro do ContentHeader pra já ficar mais preparado.